io.github.lucasew.regex101.yaml

```diff
@@ -0,0 +1,33 @@
+id: io.github.lucasew.regex101
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-23.08
+command: io.github.lucasew.regex101
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents/regex101.sqlite
```

Comment:
this is NEVER a filename.

It should likely be stored somewhere in XDG_HOME, at worst just use `--persist`

Reply:
What is the right way then?

It only stores state in this sqlite file, I didn't want to expose the whole Documents folder because it doesn't need.

Reply:
using --filesystem=xdg-documents as --persist doesn't have these shortcuts